sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/ApplicationInsightsRestClient.cs

```diff
@@ -89,14 +89,18 @@ internal HttpMessage CreateTrackRequest(IEnumerable<TelemetryItem> body)
                 content.WriteNewLine();
             }
 
+#if DEBUG
```

Comment:
qn: What is the need of this feature at all?

Reply:
Our team finds this useful when developing features or investigating issues.
This has been in the SDK since day one. We want to remove it now as we're preparing to go GA.

Reply:
Looks very custom. Could we simply use https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/Internal/OpenTelemetrySdkEventSource.cs#L34 / similar?

Reply:
> The other DebugWriter in the RestClient is used to printing actual telemetry to console, same thing that classic Application Insights SDK does.

My question was why we need that capability to be ported to OTel AzMon Exporter? (It existed in Application Insights for some reasons, and that was a contract.)

This is not a blocker to this PR, just trying to understand why we have this in the first place!

Reply:
>  why we need that capability

There is no business reason, this feature is exclusively to support the dev team.
We find this useful to quickly inspect the generated telemetry while working locally.